kahuna/public/js/edits/service.js

```diff
@@ -149,6 +149,17 @@ service.factory('editsService',
         return existingRequestPool.promise;
     }
 
+    function perform(resource, action, originalImage) {
+        const newRequest = resource.perform(action).
+              then(action => resource.get().then(edit => getSynced(originalImage, newImage => matches(edit, newImage))));
```

Comment:
Yes I think that would make sense: return the created/altered entity, with the URI to identify it.